src/world/common/SetNpcColHeight.inc.c

```diff
@@ -0,0 +1,17 @@
+#include "common.h"
+
+static void SetNpcColHeight(ScriptInstance* script) {
```

Comment:
Same notes as for SetNpcAnim here

Reply:
This is **not a script API function**. It's called as part of `func_802403D4` (isk_04 vram), so that function should probably be included in this `.inc.c` and the function shouldn't have a CamelCase name.